projects/vanilla/src/app/app.module.ts

```diff
@@ -77,10 +77,14 @@ import { ConfiguratorsModule } from "./configurators/configurators.module";
 import { ConditionsModule } from "@sinequa/ngx-ui-builder";
 import { AppConfigService } from "./app-config.service";
 
+// standalone components
+import { SearchFormComponent } from "@sinequa/components/search-form";
+import { FiltersModule } from "@sinequa/components/filters";
+
 
 // Initialization of @sinequa/core
 export const startConfig: StartConfig = {
-    //app: "training",
+    app: "training",
```

Comment:
This comment should not be removed, or else the app won't work when packaged in the product.